travel_platform/izi_travel_client.py

```diff
@@ -82,21 +111,25 @@ def find_city_by_name(self, query):
         Returns:
             dict: City data.
         """
-        url = '{base_url}mtg/objects/search'.format(base_url=self.api_url)
+
+        url = self._prepare_base_url(self.SEARCH_ENDPOINT)
         prepared_params = {"type": "city", "query": query}
         params = self._prepare_params(**prepared_params)
-        response = requests.get(url, params, headers=self.headers)
+        response = self._make_request(url, **params)
         city = {}
-        self._check_response_status(response)
-        if response.json():
-            city_data = response.json()[0]
-            city["city_uuid"] = city_data.get("uuid")
-            # Number of tours and museums.
-            city["children_count"] = city_data["children_count"]
-            city["map"] = city_data["map"]
-            city["title"] = city_data["title"]
-            # Coordinates, country code and uuid.
-            city["location"] = city_data["location"]
+
+        if response:
```

Comment:
To be honest, extracting via `[0]` index doesn't look fine.
I guess, it would be better to use `pop` or something like this.